.travis.yml

```diff
@@ -5,5 +5,10 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+before_install:
+  - wget https://matrix.org/git/olm/snapshot/olm-2.2.0.tar.bz2
```

Comment:
Why 2.2.0? 2.2.2 is available now. https://git.matrix.org/git/olm/snapshot/olm-2.2.2.tar.bz2